docs/amber_script.md

```diff
@@ -383,7 +401,7 @@ void main() {
 }
 END  # shader
 
-BUFFER kComputeBuffer TYPE vec2<int32> SIZE 524288 FILL 0
+BUFFER kComputeBuffer DATA_TYPE vec2<int32> SIZE 524288 FILL 0
```

Comment:
This is unrelated to the REPEAT functionality.

And I think ELEM_TYPE is better than DATA_TYPE.  Because actually the DATA_TYPE is _array of_ ....

Reply:
The field is already called DATA_TYPE, this just fixes the examples to match the parser.